src/Microsoft.Management.UI.Internal/ManagementList/Common/ListOrganizer.cs

```diff
@@ -42,21 +42,15 @@ protected override void OnKeyDown(KeyEventArgs e)
 
         partial void OnSelectItemExecutedImplementation(ExecutedRoutedEventArgs e)
         {
-            if (e.Parameter == null)
-            {
-                throw new ArgumentException("e.Parameter is null", "e");
-            }
+            ArgumentNullException.ThrowIfNull(e.Parameter);
```

Comment:
This is not a correct change, the parameter name used by `ArgumentNullException.ThrowIfNull(e.Parameter)` will be `e.Paraemter`, which is wrong.

Reply:
Example
https://source.dot.net/#Microsoft.AspNetCore.Http.Extensions/ProblemDetailsService.cs,29

Reply:
Is that the only place you find using this API in this way?

Looking through the usage here: https://source.dot.net/#System.Private.CoreLib/src/libraries/System.Private.CoreLib/src/System/ArgumentNullException.cs,432088dc6d98b845,references, I don't think we can call that an adopted pattern in .NET code.

Reply:
The original `ArgumentException` is more suitable than using `ArgumentNullException.ThrowIfNull(e.Parameter)` in this case, though it's not using a resource string for its message.

Reply:
Yes, you're right. For some reason I was sure they had changed the message template.
But it seems the original message is worst :-) 
```powershell
[argumentnullexception]::new("e.Parameter is null", "e")

Message        : e (Parameter 'e.Parameter is null')
ParamName      : e.Parameter is null
```

Reply:
The original one is using `ArgumentException`, not `argumentnullexception`.

This is how it looks:
![image](https://user-images.githubusercontent.com/127450/222538604-965ae642-7ccf-4075-a672-5b5876df5c96.png)